src/ActionList.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
+import dragula from 'react-dragula';
```

Comment:
I tried `react-sortable-hoc` first, but it requires too much changes here, and there were some issues. Didn't notice any issues with `react-dragula`. It allows us easy to switch draggable feature on/off and also we will be able to [duplicate actions](https://github.com/bevacqua/dragula/issues/359#issuecomment-270100907) in future.

Those classes are added only while dragging. We could just set `draggableActions` prop to `false` by default, so it will have no impact when not enabled.

However, if you think this feature is not needed here, we could just add a prop to pass a custom `ActionList` component. It would also allow for better customizations. For example, I was planning to add props to hide buttons since we'll not need then in the extension.

We could just use a fork for the extension if you think the purposes are different, and you want to keep the monitor simple as is. For example, using [`react-split-pane`](https://github.com/tomkp/react-split-pane) per #62, will also add global classes, and even worse prefixed. It is not a problem for the extension, but could be for Redux DevTools. Let me know what you think would be the best.